test/models/validatable_test.rb

```diff
@@ -64,7 +64,7 @@ class ValidatableTest < ActiveSupport::TestCase
     user = create_user
 
     user.password = ''
-    user.password_confirmation = ''
+    user.password_confirmation = 'x'
```

Comment:
Yes, that's because blank string doesn't trigger password update.
And it doesn't trigger validation any longer.
So one of fields had to have ```present? == true``` to trigger validations.